src/WebDriver.php

```diff
@@ -1202,7 +1202,7 @@ public function evaluateScript($script)
 
     public function wait($timeout, $condition)
     {
-        $seconds = (int) ($timeout / 1000.0);
+        $seconds = $timeout / 1000.0;
```

Comment:
`wait` accepts seconds:
https://github.com/php-webdriver/php-webdriver/blob/a1578689290055586f1ee51eaf0ec9d52895bb6d/lib/WebDriver.php#L88

Reply:
I opened this PR because this is actually breaking some of our tests, we were calling `wait(300)` and now it rounds it down to 0, so there is no wait. We can fix it using `wait(1000)`, although that will unfortunately make our tests a lot slower. I am aware that we shouldn't be using waits too much, but we're doing it at the moment because of some constraints in our project.

In any case, if you don't want to revert this that's fine I guess, feel free to close this PR. But I think it should at least be documented as a breaking change in the [release notes](https://github.com/oleg-andreyev/MinkPhpWebDriver/releases/tag/v1.3.0) (and ideally in the [CHANGELOG](https://github.com/oleg-andreyev/MinkPhpWebDriver/blob/1.3/CHANGELOG.md) as well, although it seems that hasn't been updated for a while).